redaxo/src/core/lib/setup/setup.php

```diff
@@ -222,15 +222,15 @@ public static function checkDbSecurity()
                 $security[] = rex_i18n::msg('setup_security_deprecated_mariadb', $dbVersion);
             } elseif (1 == version_compare($dbVersion, '10.6', '<') && time() > strtotime('1 Jun 2025')) {
                 $security[] = rex_i18n::msg('setup_security_deprecated_mariadb', $dbVersion);
-            } elseif (1 == version_compare($dbVersion, '10.7', '<') && time() > strtotime('1 Jul 2026')) { // LTS
+            } elseif (1 == version_compare($dbVersion, '10.7', '<') && time() > strtotime('1 Feb 2023')) { // LTS
                 $security[] = rex_i18n::msg('setup_security_deprecated_mariadb', $dbVersion);
-            } elseif (1 == version_compare($dbVersion, '10.8', '<') && time() > strtotime('1 Feb 2023')) {
+            } elseif (1 == version_compare($dbVersion, '10.8', '<') && time() > strtotime('1 May 2023')) {
```

Comment:
Die Daten sind jetzt alle eins verrutscht.
Hier in der Zeile muss zum Beispiel das Enddatum von 10.7 hin, denn der Check vorne ist ja `< 10.8`.
Also wenn Version kleiner 10.8, und das Endddatum von 10.7 (Feb 2023) erreicht, dann Warnung.

Somit stimmten die Daten schon alle, die hier notiert waren.
Aber es besteht eben einfach ein Logikproblem, da die Enddaten nicht chronologisch entsprechend der Versionsnummern sind.

Somit war der erste Verdacht, dass einfach nur die Daten falsch bei uns hinterlegt sind, wohl falsch.
Daher können wir den PR hier so glaube ich dann doch schließen. Oder magst du dich an der Logikänderung versuchen?

Reply:
Ich versuche mich mal an einer Logik-Änderung. Heute klappt das nicht mehr. Die Tage dann.
Sofern eine:r von Euch eine gute Idee hat, gern melden.